src/po/page/base.page.js

```diff
@@ -1,11 +1,26 @@
-import { BasePageComponents } from "../components/index.js";
+import { BasePageComponents, HeaderComponent } from "../components/index.js";
+import {assert} from "chai";
 
 export class Basepage {
   constructor() {
     this.basePageComponents = new BasePageComponents();
+    this.headerComponent = new HeaderComponent();
   }
 
   async open() {
     await browser.url("https://trello.com/");
   }
+  async userlogInSuccessfully () {
```

Comment:
Probably just 
login(user, password)
we should put pass and userName as params as there can be a lot of testing accounts
and think about moving assertions to test level